<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Text de-toxification</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.jpg</normaloff>icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Visualizations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1081</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="exploratory_tab">
     <attribute name="title">
      <string>Exploratory</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../reports/figures/toxic_texts_word_cloud.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>431</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../reports/figures/non-toxic_texts_word_cloud.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>300</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../reports/figures/distribution_reference_toxicity_scores.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>290</y>
        <width>300</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../reports/figures/distribution_translated_toxicity_scores.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>10</y>
        <width>300</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../reports/figures/distribution_length_differences.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>290</y>
        <width>300</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../reports/figures/distribution_similarity_scores.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="results_tab">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Toxic sentence:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_toxic">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>841</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>I like that shit.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_non_toxic">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>841</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>I like it.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Non-toxic sentence:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="detoxification_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>1041</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Detoxification</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_toxity_result">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>1031</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
